dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolver.java

```diff
@@ -543,42 +692,49 @@ List<List<RepositoryResource>> createInstallLists() {
      * @return the ordered list of resources to install
      */
     List<RepositoryResource> createInstallList(SampleResource resource) {
```

Comment:
Rename this and similar methods to indicate that they also record missing dependencies.